app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -34,7 +34,6 @@
 import edu.grinnell.kdic.visualizer.VisualizeFragment;
 
 public class MainActivity extends AppCompatActivity {
-    //Set the tag for Main Activity class
     public static final String TAG = MainActivity.class.getSimpleName();
```

Comment:
Should this tag be public? if you're not using it through out the app, it should stay private